content/child/resource_dispatcher.cc

```diff
@@ -261,6 +269,12 @@ void ResourceDispatcher::OnReceivedData(int request_id,
   TRACE_EVENT0("loader", "ResourceDispatcher::OnReceivedData");
   DCHECK_GT(data_length, 0);
   PendingRequestInfo* request_info = GetPendingRequestInfo(request_id);
+#if defined(NFS_SHARED_MEMORY)
+  // Seek the shared memory file to trigger nfs server -> client synching.
+  FILE* fp = fdopen(request_info->buffer->handle().GetHandle(), "rw");
+  fseek(fp, 0L, SEEK_END);
+#endif
```

Comment:
Yes, It will be helpful to use on other modules. Shall we add a new file as below ?

// Copyright 2018 Samsung Electronics. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#ifndef BASE_MEMORY_SHARED_MEMORY_CASTANETS_HELPER_H_
#define BASE_MEMORY_SHARED_MEMORY_CASTANETS_HELPER_H_

namespace base {

namespace nfs_util {

#if defined(NFS_SHARED_MEMORY)
// This method flushes the changes on memory mapped region to
// the underlying network file system.
inline void FlushToDisk(int fd) {
  fdatasync(fd);
}

// Temporary workaround to get nfs server->nfs client updates
// synched.
inline void Sync(int fd) {
  FILE* fp = fdopen(fd, "rw");
  fseek(fp, 0L, SEEK_END);
}
#endif

}  // namespace nfs_util

}  // namespace base

#endif  // BASE_MEMORY_SHARED_MEMORY_CASTANETS_HELPER_H_